test/unit/model/include.test.js

```diff
@@ -208,10 +208,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           include: [{ where: { active: false }, model: this.Project.scope('that'), as: 'projects' }],
         });
 
-        // TODO [chai@>5]: simplify once '.deep.equals' includes support for symbols (https://github.com/chaijs/chai/issues/1054)
-        expect(options.include[0]).to.have.property('where');
-        expect(Utils.getComplexKeys(options.include[0].where)).to.deep.equal([Op.and]);
-        expect(options.include[0].where[Op.and]).to.deep.equal([{ that: false }, { active: false }]);
+        expect(options.include[0].where).to.deep.equal({ [Op.and]: [{ that: false }, { active: false }] });
```

Comment:
@ephys is this what you had in mind for this test?

Reply:
that sounds correct to me yes